### PR TITLE
about screen layout orientation changes

### PR DESCRIPTION
**Pull request by V9vek**
The guy who added the About Screen didn't add the `NestedScrollView` for the layout changes when we change to landscape orientation

`Previous About Screen:`
prev_about

`After About Screen I have made changes:`

https://user-images.githubusercontent.com/40730402/136022005-95bf407a-ba46-47dc-bb93-2615dca73ae1.mp4

### CODE REVIEW

**Comment by V9vek** [comment]
I don't know why the video is in pink 😆 
But please review and check on your device and do let me know

**Comment by rishuyadav** [comment]
Thats gr8 @V9vek 👍🏻can u please create an issue also regarding this, then we can merge ur pr with reference to that issue, plus u get points for issues also🚀

**Comment by gaganmalvi** [comment]
Video works fine here, good job. Please create an issue and reference it here, we'll merge it.

**Comment by V9vek** [comment]
I have raised a issue regarding this, please check

File: app/src/main/res/layout/activity_about.xml [hunk 1]
```diff
@@ -74,45 +50,75 @@
...
+            <TextView
+                android:id="@+id/textView5"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="16dp"
+                android:layout_marginTop="32dp"
+                android:layout_marginEnd="16dp"
+                android:fontFamily="@font/poppins"
+                android:gravity="center"
+                android:text="ACM-VIT, one of the most reputed and distinguished technical chapter in VIT-Vellore has been working with vision and determination since it’s inception in 2009."
```
Comment (rishuyadav) [inline app/src/main/res/layout/activity_about.xml:105]: This string can be extracted from string res

Comment (V9vek) [reply app/src/main/res/layout/activity_about.xml:105]: okay I am doing that and will push changes
anything else?

**Comment by rishuyadav** [comment]
Hey @V9vek pls save strings and extract from there, then the pr is good to merge 👍

File: app/src/main/res/layout/activity_about.xml (hunk 1 above)
Comment (rishuyadav) [reply app/src/main/res/layout/activity_about.xml:105]: Nope nothing else, just extract all hardcoded strings

**Comment by V9vek** [comment]
Removed hardcoded strings and extracted from string res file 
Please check it out and merge it with the required `hacktoberfest` tags

Thanks 🙏🏼

**Comment by rishuyadav** [comment]
Thanks a lot @V9vek for contributing 
Happy Hacktoberfest 🥳

**Pull request merged**